content/source/docs/enterprise-beta/users-teams-organizations/index.html.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: enterprise2
+page_title: "Users, Teams, and Organizations - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-users-teams-organizations"
+---
+
+# Users, Teams, and Organizations
+
+This documentation explains the organizational and access control model in
+Terraform Enterprise.
+
+**Users** are individual members of an organization. They belong to teams which
+are granted permissions to an organization’s workspaces. Users can also be
+added directly to a workspace by being added as a collaborator.
```

Comment:
I'd cut out "Users can also be added directly to a workspace by being added as a collaborator." per the slack convo